test/vm.install

```diff
@@ -17,6 +17,10 @@ if ! rpm -q beakerlib; then
     fi
 fi
 
+# disable mirrors & re-enable direct download
+sed -i "s/metalink.*//" /etc/yum.repos.d/*.repo
+sed -i "s/#\(baseurl=.*\)/\1/" /etc/yum.repos.d/*.repo
```

Comment:
That's a good start, but on Fedora at least, the url they have in there by default is backed by some sort of random/round-robin/load-balancing DNS so you still don't get the same server every time. When I really want the exact same thing I use dl.fedoraproject.org instead of download.fedoraproject.org